livingdoc-engine/src/main/kotlin/org/livingdoc/engine/execution/examples/scenarios/matching/ScenarioFixtureChecker.kt

```diff
@@ -0,0 +1,103 @@
+package org.livingdoc.engine.execution.examples.scenarios.matching
+
+import org.livingdoc.api.fixtures.scenarios.After
+import org.livingdoc.api.fixtures.scenarios.Before
+import org.livingdoc.api.fixtures.scenarios.Binding
+import org.livingdoc.api.fixtures.scenarios.Step
+import org.livingdoc.engine.execution.examples.scenarios.ScenarioFixtureModel
+import java.lang.reflect.Method
+import java.lang.reflect.Modifier
+import kotlin.reflect.KClass
+
+
+internal object ScenarioFixtureChecker {
```

Comment:
Is not used yet and corresponding test does not include any tests for this class since it is not actually used...

Reply:
@slu-it, the checker is actually used in `ScenarioExecution.kt:45`, and there is a `ScenarioFixtureCheckerTest`. Am I missing something here?